### DISCUSSION TITLE
ALPACA IS ON THE MAIN PAGE OF FLATHUB

### DISCUSSION THREAD

**Discussion by Jeffser** [Jeffser/Alpaca#169] [Show and tell]
I'm not sure how Flathub chooses which apps to highlight on the main page, but Alpaca is the first app that appears!

I couldn't be happier, just seeing something I've made be promoted here is really really awesome to me.

![image](https://github.com/user-attachments/assets/f433c2be-c72c-4daa-9df6-626a3f4c8552)

As always thanks to all of you for your help, contributions and support!

**Comment by loulou64490** [comment]
Wooooow congrulations !!!
